mpdev/BUILD.bazel

```diff
@@ -12,4 +12,4 @@ go_binary(
     name = "mpdev",
     embed = [":go_default_library"],
     visibility = ["//visibility:public"],
-)
```

Comment:
We should keep the newlines at end of file--here and others

Reply:
Oops done